cosmos/stride.json

```diff
@@ -99,63 +113,152 @@
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/stride/stusomm.png"
     }
   ],
-  "feeCurrencies": [
-    {
-      "coinDenom": "STRD",
-      "coinMinimalDenom": "ustrd",
-      "coinDecimals": 6,
-      "coinGeckoId": "stride",
-      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/stride/ustrd.png",
-      "gasPriceStep": {
-        "low": 0.0005,
-        "average": 0.005,
-        "high": 0.05
-      }
-    },
-    {
-      "coinDenom": "ATOM",
-      "coinMinimalDenom": "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2",
-      "coinDecimals": 6,
-      "coinGeckoId": "cosmos",
-      "gasPriceStep": {
-        "low": 0.0001,
-        "average": 0.001,
-        "high": 0.01
-      }
-    },
-    {
-      "coinDenom": "OSMO",
-      "coinMinimalDenom": "ibc/D24B4564BCD51D3D02D9987D92571EAC5915676A9BD6D9B0C1D0254CB8A5EA34",
-      "coinDecimals": 6,
-      "coinGeckoId": "osmosis",
-      "gasPriceStep": {
-        "low": 0.001,
-        "average": 0.01,
-        "high": 0.1
+    "feeCurrencies": [
+      {
+        "coinDenom": "STRD",
+        "coinMinimalDenom": "ustrd",
+        "coinDecimals": 6,
+        "coinGeckoId": "stride",
+        "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/stride/ustrd.png",
+        "gasPriceStep": {
+          "low": 0.0005,
+          "average": 0.005,
+          "high": 0.05
+        }
+      },
+      {
+        "coinDenom": "ATOM",
+        "coinMinimalDenom": "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2",
+        "coinDecimals": 6,
+        "coinGeckoId": "cosmos",
+        "gasPriceStep": {
+          "low": 0.0001,
+          "average": 0.001,
+          "high": 0.01
+        }
+      },
+      {
+        "coinDenom": "OSMO",
+        "coinMinimalDenom": "ibc/D24B4564BCD51D3D02D9987D92571EAC5915676A9BD6D9B0C1D0254CB8A5EA34",
+        "coinDecimals": 6,
+        "coinGeckoId": "osmosis",
+        "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/osmosis/uosmo.png",
```

Comment:
@riley-stride  You can only add coinImageUrls that are in the identifier of that chain.

Please add `uosmo.png` to the `images/stride` directory and modify the path.

Reply:
done